stable/traefik/templates/NOTES.txt

```diff
@@ -19,6 +19,10 @@
 
      http - {{ .Values.service.nodePorts.http }}
      https - {{ .Values.service.nodePorts.https }}
+     {{- if eq .Values.dashboard.serviceType "NodePort" }}
+     dashboard - {{ .Values.dashboard.nodePorts.http }}
```

Comment:
Do you need to check that the value of .Values.dashboard.nodePorts.http is not empty here as well?